tests/Microsoft.Bot.Builder.Teams.Tests/TeamsActivityHandlerTests.cs

```diff
@@ -793,10 +794,10 @@ protected override Task OnTeamsMessagingExtensionCardButtonClickedAsync(ITurnCon
                 return base.OnTeamsMessagingExtensionCardButtonClickedAsync(turnContext, cancellationToken);
             }
 
-            protected override Task<MessagingExtensionActionResponse> OnTeamsMessagingExtensionFetchTaskAsync(ITurnContext<IInvokeActivity> turnContext, CancellationToken cancellationToken)
+            protected override Task<MessagingExtensionActionResponse> OnTeamsMessagingExtensionFetchTaskAsync(ITurnContext<IInvokeActivity> turnContext, MessagingExtensionQuery query, CancellationToken cancellationToken)
             {
                 Record.Add(MethodBase.GetCurrentMethod().Name);
-                return base.OnTeamsMessagingExtensionFetchTaskAsync(turnContext, cancellationToken);
+                return base.OnTeamsMessagingExtensionFetchTaskAsync(turnContext, query, cancellationToken);
```

Comment:
my bad... these tests could perhaps be a little deeper :-(
(they only check the dispatching not the values in the query)